### DISCUSSION TITLE
CoPilot

### DISCUSSION THREAD

**Discussion by Trigentblr** [community/community#110274] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

How to purchase the Copilot and assign them to the users?

**Comment by davecheney** [comment, marked as answer]
@Trigentblr this is the Copilot for Business, or recently announced Copilot for Enterprise, product. https://docs.github.com/en/enterprise-cloud@latest/copilot/copilot-business/about-github-copilot-business
